model/database/registry.yaml

```
@@ -542,7 +543,9 @@ groups:
               value: "ConsistentPrefix"
               stability: experimental
         stability: experimental
-        brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+        brief: >
+          Effective Request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+          This represents the first applicable consistency level among those specified by the request, the client or the account's default consistency level.
```

```suggestion
          This represents the first applicable consistency level among those specified by the request, the client's default consistency level, or the account's default consistency level.
```

---

it is not client default consistency, consistency is defined at client level also

---

I found the sentence confusing, is this correct then?

```suggestion
          This represents the first applicable consistency level among those specified by the request, the client's consistency level or the account's default consistency level.
```

---

updated to this "This represents the applicable consistency level determined based on the hierarchy of settings: request-level, client instance-level, and account-level default consistency levels."